content/css/concepts/typography/terms/font-style/font-style.md

```diff
@@ -7,13 +7,18 @@ Subjects:
 Tags:
   - 'Fonts'
   - 'Typography'
+  - 'Style'
 CatalogContent:
   - 'learn-css'
   - 'paths/front-end-engineer-career-path'
   - 'paths/full-stack-engineer-career-path'
 ---
 
-To set the font style in which text will appear.
+Changing the **font-style** selection will determine if the font is vertical or has a degree of slant. The most commonly used varient of font-style is `italic`.
+
+![font-styles in html.](https://github.com/SarahCarlock/codecademyDocs/assets/112113100/4dce07d9-bd71-447c-94c6-d8468d66e150)
+
+![font-styles visual examples.](https://github.com/SarahCarlock/codecademyDocs/assets/112113100/fcb3817f-86ac-47f5-a44e-386da8ad439a) 
```

Comment:
For this do the following things
- Move this to the end of the entry after all the examples.
- Change the link to this format: `https://raw.githubusercontent.com/Codecademy/docs/main/media/` followed by the image name. For this, you'll have to add your image to the media folder on this branch.

Reply:
Yep!